File: Java/UberPool.java
package Java;

public class UberPool extends UberCar {

    String brand;
    String model;

    public UberPool(String license, Account driver, Integer passenger, String brand, String model) {
        //Use super to pass parameter to subclass (Using Inheritance)
        super(license, driver, passenger);
        this.brand = brand;
        this.model = model;
    }
    
}

File: PHP/UberPool.php
<?php
    require_once 'UberCar.php';
    //How to use Inheritance in PHP
    class UberPool extends UberCar{
        public $brand;
        public $model;

        public function __construct($license, $driver, $brand, $model){
            //To pass parameter to construct use parent
            parent::__construct($license,$driver);
            $this->brand = $brand;
            $this->model = $model;
        }
    }
?>